packages/beacon-node/src/network/peers/score/score.ts

```diff
@@ -46,6 +48,30 @@ export class RealScore implements IPeerScore {
     this.setLodestarScore(newScore);
   }
 
+  // TODO: @twoeths these times need some massaging.  Curious to get your thoughts
+  applyReconnectionCoolDown(reason: GoodByeReasonCode): number {
+    let coolDownMin = NO_COOL_DOWN_APPLIED;
+    switch (reason) {
+      // let scoring system handle score decay by itself
+      case GoodByeReasonCode.BANNED:
+      case GoodByeReasonCode.SCORE_TOO_LOW:
+        break;
+      case GoodByeReasonCode.INBOUND_DISCONNECT_NO_GOODBYE:
+      case GoodByeReasonCode.TOO_MANY_PEERS:
+        coolDownMin = 5;
+        break;
+      case GoodByeReasonCode.ERROR:
+      case GoodByeReasonCode.CLIENT_SHUTDOWN:
+        coolDownMin = 60;
+        break;
+      case GoodByeReasonCode.IRRELEVANT_NETWORK:
+      // should this just remove the peer from the peerManager?
+    }
+    // set banning period to time in ms in the future from now
+    this.lastUpdate = Date.now() + coolDownMin * 60 * 1000;
```

Comment:
if `NO_COOL_DOWN_APPLIED = -1` it has side effect here, any down side to set this constant to 0?